mux_test.go

```diff
@@ -327,6 +365,9 @@ func TestMux(t *testing.T) {
 	// Expect a settle signal for node 2.
 	expectResponse(<-l[1].responseChan, 2, routerrpc.ResolveHoldForwardAction_SETTLE)
 
+	// Notify final htlc resolution for htlc 2 on node 2.
+	l[1].notifyFinal(2)
+
 	// Wait for invoice-level settle signal.
 	settledChan := make(chan struct{})
 	go func() {
```

Comment:
I saw one CI test failure. Timeout on this goroutine.

Reply:
Turned out to be a test problem with a `time.Sleep` that wasn't long enough. Replaced by a proper wait for signal.